### PR TITLE
Fixing order of items in the table

### PR DESCRIPTION
**Pull request by RossAdams**
Order of items was incorrect fixing that so that user and domain impersonation come before spam and swapping their order.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/office-365-security/how-policies-and-protections-are-combined.md [hunk 1]
```diff
@@ -39,10 +39,10 @@ There are two major factors that determine which policy is applied to a message:
...
   |4|Spoofing|CAT:SPOOF|[Configure spoof intelligence in EOP](learn-about-spoof-intelligence.md)|
-  |5|Spam|CAT:SPM|[Configure anti-spam policies in EOP](configure-your-spam-filter-policies.md)|
-  |6|Bulk|CAT:BULK|[Configure anti-spam policies in EOP](configure-your-spam-filter-policies.md)|
-  |7<sup>\*</sup>|Domain impersonation (protected users)|DIMP|[Configure ATP anti-phishing policies](configure-atp-anti-phishing-policies.md)|
-  |8<sup>\*</sup>|User impersonation (protected domains)|UIMP|[Configure ATP anti-phishing policies](configure-atp-anti-phishing-policies.md)|
+  |5<sup>\*</sup>|User impersonation (protected domains)|UIMP|[Configure ATP anti-phishing policies](configure-atp-anti-phishing-policies.md)|
+  |6<sup>\*</sup>|Domain impersonation (protected users)|DIMP|[Configure ATP anti-phishing policies](configure-atp-anti-phishing-policies.md)|
+  |7|Spam|CAT:SPM|[Configure anti-spam policies in EOP](configure-your-spam-filter-policies.md)|
+  |8|Bulk|CAT:BULK|[Configure anti-spam policies in EOP](configure-your-spam-filter-policies.md)|
   |
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/how-policies-and-protections-are-combined.md:46]:
```suggestion
```

Comment (JohanFreelancer9) [reply microsoft-365/security/office-365-security/how-policies-and-protections-are-combined.md:46]: Please also remove blank line 47.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@RossAdams Please update the document with suggested changes. Thanks

**Comment by RossAdams** [comment]
> @RossAdams Please update the document with suggested changes. Thanks

Not sure what you are asking me to do here.  I already updated the doc?

**Review by chrisda — APPROVED** [review]

**Pull request merged**